python/mleap/sklearn/logistic.py

```diff
@@ -75,12 +75,23 @@ def serialize_to_bundle(self, transformer, path, model_name):
         inputs = [{
                   "name": transformer.input_features,
                   "port": "features"
-                }]
-
-        outputs = [{
-                  "name": transformer.prediction_column,
-                  "port": "prediction"
-                }]
+                  }]
+
+        outputs = list()
+        outputs.append({
+            "name": transformer.prediction_column,
+            "port": "prediction"
+        })
+
+        outputs.append({
+            "name": "raw_prediction",
+            "port": "raw_prediction"
+        })
+
+        outputs.append({
+            "name": "probability",
```

Comment:
Could we make the name of the raw_prediction and probability columns configurable in the mleap_init function? Also maybe allow for these to be null as default, in which case they wouldn't be added to the outputs list. What do you think?

Reply:
What about `transformer.prediction_column} + "_raw"` and `transformer.prediction_column + "_proba"`for the `name` keys of these outputs, respectively?

Reply:
I would like to still keep the option of generating these additional outputs only when specified to do, it would be similar to how Spark handles this as well.

Reply:
Let me know if you need any help to make these changes, happy to help.